test/act-mapping/aria-props-allowed.json

```diff
@@ -0,0 +1,5 @@
+{
+  "id": "5c01ea",
```

Comment:
Question: how do we want to name these files? We don't appear to have a naming convention for axe-core, and I'm not sure where the `aria-props-allowed` name comes from. Is that what it's called in ACT (but not listed in the rule itself)?

Reply:
I was sort of just winging it. ACT rules do have a file name. I could copy those I suppose.